tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -259,6 +259,33 @@ public async Task AcquireToken_ManagedUsernameIncorrectPassword_AcquireTokenSucc
             Assert.Fail("Bad exception or no exception thrown");
         }
 
+        [TestMethod]
+        [TestCategory(TestCategories.ADFS)]
+        public async Task AcquireTokenRefreshInTestAsync()
+        {
+            LabResponse labResponse = await LabUserHelper.GetSpecificUserAsync("fIDLABOC_Refresh_IN@MSIDLAB6.com").ConfigureAwait(false);

```

Comment:
@bgavrilMS I tried to use this for testing the refresh in value but it is not returning the value in the auth response. I there something special that I have to do?

Reply:
I'm pretty sure ADFS doesn't implement refresh_in. 

for testing I would recommend focusing more on unit tests. 

1. add assertions to the tests in RefreshInTests
2. find some refresh RT tests that refresh the RT because the AT expired or because withforcerefresh is used.
3. etc.

Reply:
I tried this with the cloud account as well and it still didn't work. perhaps the policies are not setup correctly on the lab account